source/psp/video_hardware_main.cpp

```diff
@@ -914,14 +914,93 @@ int	lastposenum;
 float old_i_model_transform;
 //
 
+void GL_DrawAliasBlendedWireFrame (aliashdr_t *paliashdr, int pose1, int pose2, float blend)
```

Comment:
I moved this to its own function to get rid of the per primitive checks for GU_LINE_STRIP, so it's now checked only per model not per prim.